requirements.txt

```diff
@@ -8,7 +8,7 @@ click==8.1.3
 craft-cli==1.1.0
 craft-grammar==1.1.1
 craft-parts==1.10.2
-craft-providers==1.3.1
+git+https://github.com/canonical/craft-providers#egg=craft-providers
```

Comment:
Let's release craft-providers, I need it too.

Reply:
`craft-providers` is released!